cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -152,6 +152,25 @@ size_t batched_decompress_temp_size(compression_type compression,
   return temp_size;
 }
 
+void check_is_zstd_enabled()
+{
+  CUDF_EXPECTS(NVCOMP_HAS_ZSTD_DECOMP, "nvCOMP 2.3 or newer is required for Zstandard compression");
+  CUDF_EXPECTS(NVCOMP_ZSTD_IS_STABLE or cudf::io::detail::nvcomp_integration::is_all_enabled(),
+               "Zstandard compression is experimental, you can enable it through "
+               "`LIBCUDF_NVCOMP_POLICY` environment variable.");
+
+#if NVCOMP_ZSTD_IS_DISABLED_ON_PASCAL
```

Comment:
I don't see this being defined. Are we going to do that in a separate PR?

Reply:
ah, I accidentally removed the define before pushing the change. Will fix, thanks.

Reply:
should be correct now